snippets/pipeLog.md

```diff
@@ -0,0 +1,13 @@
+### pipeLog
+
+Use `console.log` in a pipeline as this function simply encloses it and returns the passed value. This is especially useful for debugging when you want to log a variable's value before its usage.
```

Comment:
This should be expanded to explain the how of this snippets inner workings and be less about use-cases and more about the details of how the code works (although there is very little to see here anyways). It should also come after the functionality description.